Header header
int32 position

uint8 command
uint32 force_threshold
